modules/script/script.sh

```diff
@@ -3,13 +3,13 @@
 # Tell build process to exit if there are any errors.
 set -oue pipefail
 
-get_yaml_array RUN '.run[]' "$1"
+get_yaml_array SCRIPTS '.scripts[]' "$1"
 
 cd "$CONFIG_DIRECTORY/scripts"
 
 find "$PWD" -type f -exec chmod +x {} \;
 
-for CMD in "${RUN[@]}"; do
-    echo "Running command: $CMD"
-    eval "$CMD"
+for SCRIPT in "${SCRIPTS[@]}"; do
```

Comment:
this thing just is not working????? 
no matter how I put it, it [can't find the script files](https://github.com/ublue-os/startingpoint/actions/runs/5848195554/job/15855225241) when trying to run them, but [can find them with `ls`](https://github.com/ublue-os/startingpoint/actions/runs/5848141493/job/15855113964).

time to take a break...